src/includes/topics/template.php

```diff
@@ -1600,6 +1599,11 @@ function bbp_get_topic_author_role( $args = array() ) {
 
 		$topic_id = bbp_get_topic_id( $r['topic_id'] );
 		$role     = bbp_get_user_display_role( bbp_get_topic_author_id( $topic_id ) );
+		$css_role = sanitize_key( $role );
+		
+		if( empty( $args['before'] ) ) {
```

Comment:
```suggestion
		if ( empty( $args['before'] ) ) {
```

Reply:
Good catch @robinwpdeveloper vai please do the changes according to WordPress coding standards.